src/session/session.js

```diff
@@ -0,0 +1,29 @@
+import { SAVE_SESSION, LOG_OUT } from "./actions";
+
+const initialState = {
+  user: undefined,
+  authToken: undefined
+};
+
+export function sessionReducer(state = initialState, action) {
```

Comment:
The usage of different session reducers combined let us split the modules responsibilities easily.